app/components/task/holder.js

```diff
@@ -14,6 +14,20 @@ export default class TasksHolderComponent extends Component {
   TASK_KEYS = TASK_KEYS;
   availabletaskStatusList = TASK_STATUS_LIST;
 
+  constructor() {
+    super(...arguments);
+    if (this.args.dev) {
+      this.status =
+        this.args.task.status === TASK_KEYS.COMPLETED
+          ? TASK_KEYS.DONE
+          : this.args.task.status;
+    } else {
+      this.status =
+        this.args.task.status === TASK_KEYS.DONE
```

Comment:
So just to confirm we are migrating all the `COMPLETED` statuses to `DONE`?

Reply:
In some states where something is done and others are completed, only one would show.

Reply:
> So just to confirm we are migrating all the `COMPLETED` statuses to `DONE`?

Yes

Reply:
> In some states where something is done and others are completed, only one would show.

Yes. Reason being with dev mode enabled we have only `DONE` status and without dev mode being enabled we only consider `COMPLETED`.

Reply:
Understood